packages/reporter/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { gnosis, goerli, mainnet } from "viem/chains"
```

Comment:
To make the import convention more specific, we can create detailed instructions:

* Start with external imports.
* Add a blank line.
* Then, add internal imports.

External Imports:

```js
import { goerli, gnosis, mainnet } from "viem/chains";
import "dotenv/config";
import winston from "winston";
```

Internal Imports:

```js
import AMBController from "./controller/AMBController";
import BlocksListener from "./BlockListener";
import Multiclient from "./MultiClient";
import SygmaController from "./controller/SygmaController";
import TelepathyController from "./controller/TelepathyController";
```

Use this rule everywhere in the project.